code/modules/organs/organ_internal.dm

```diff
@@ -168,13 +168,24 @@
 				INTERNAL ORGANS TYPES
 ****************************************************/
 
-/datum/internal_organ/heart // This is not set to vital because death immediately occurs in blood.dm if it is removed.
+/datum/internal_organ/heart // This is not set to vital because death immediately occurs in blood.dm if it is removed. Also, all damage effects are handled there.
 	name = "heart"
 	parent_limb = "chest"
 	removed_type = /obj/item/organ/heart
 	robotic_type = /obj/item/organ/heart/prosthetic
 	organ_id = ORGAN_HEART
 
+/datum/internal_organ/heart/process()
+	..()
+	
+	if(!owner.reagents.get_reagent_amount(/datum/reagent/medicine/peridaxon) >= 0.05)
+		if(is_bruised())
+			if(prob(5))
+				spawn owner.emote("me", 1, "grabs at his chest!")
+		else if(is_broken())
+			if(prob(20))
```

Comment:
you can concatenate the if together (is_broken && prob(20))

Reply:
will do